[2023-06-03T20:32:11.967+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-02T00:00:00+00:00/task_id=run_after_loop permission to 509
[2023-06-03T20:32:11.968+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-02T00:00:00+00:00/task_id=run_after_loop permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-02T00:00:00+00:00/task_id=run_after_loop'
[2023-06-03T20:32:11.992+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-02T00:00:00+00:00/task_id=run_after_loop permission to 509
[2023-06-03T20:32:11.993+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-02T00:00:00+00:00/task_id=run_after_loop permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-02T00:00:00+00:00/task_id=run_after_loop'
[2023-06-03T20:32:12.023+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_bash_operator.run_after_loop scheduled__2023-06-02T00:00:00+00:00 [queued]>
[2023-06-03T20:32:12.030+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_bash_operator.run_after_loop scheduled__2023-06-02T00:00:00+00:00 [queued]>
[2023-06-03T20:32:12.031+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-03T20:32:12.042+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): run_after_loop> on 2023-06-02 00:00:00+00:00
[2023-06-03T20:32:12.048+0000] {standard_task_runner.py:57} INFO - Started process 3577 to run task
[2023-06-03T20:32:12.050+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', 'scheduled__2023-06-02T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmp44e2os3g']
[2023-06-03T20:32:12.052+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask run_after_loop
[2023-06-03T20:32:12.096+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-02T00:00:00+00:00/task_id=run_after_loop permission to 509
[2023-06-03T20:32:12.096+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-02T00:00:00+00:00/task_id=run_after_loop permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-02T00:00:00+00:00/task_id=run_after_loop'
[2023-06-03T20:32:12.101+0000] {task_command.py:410} INFO - Running <TaskInstance: example_bash_operator.run_after_loop scheduled__2023-06-02T00:00:00+00:00 [running]> on host cd81537d5f1e
[2023-06-03T20:32:12.162+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_bash_operator' AIRFLOW_CTX_TASK_ID='run_after_loop' AIRFLOW_CTX_EXECUTION_DATE='2023-06-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-02T00:00:00+00:00'
[2023-06-03T20:32:12.168+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-03T20:32:12.170+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo 1']
[2023-06-03T20:32:12.180+0000] {subprocess.py:86} INFO - Output:
[2023-06-03T20:32:12.184+0000] {subprocess.py:93} INFO - 1
[2023-06-03T20:32:12.186+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-03T20:32:12.208+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20230602T000000, start_date=20230603T203212, end_date=20230603T203212
[2023-06-03T20:32:12.223+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-03T20:32:12.237+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
